### PR TITLE
proposed changes to Chapter 2

### PR DESCRIPTION
**Pull request by bhs**
This is a messy diff... a combination of what was there before and some net new content. Probably best to just read the new version and comment or propose any and all changes - I am not attached to this!

### CODE REVIEW

File: 02 - customers-and-engineers.md [hunk 1]
```diff
@@ -218,20 +220,20 @@ each step, I know the total duration. Indeed, instead of thinking about
...
-and understand these changes.
+If we extend that metaphor a bit, then the SLO describes the acceptable range
+for those transactional SLIs.
+
+It follows that the SLO becomes a contract between transactions and resources
+to define acceptable performance bounds, and thus the SLO is a contract between
+the end-users and engineers themselves. Instead of thinking about observability
+as the ability to 'explore unknown unknowns' or 'quickly reduce search space
+during incidents', we can begin to think of it as a way to define and iterate
+on SLI's and use those to formalize SLO's with each other, setting and
```
Comment (austinlparker) [inline 02 - customers-and-engineers.md:231]:
```suggestion
on SLIs and use those to formalize SLOs with each other, setting and
```

**Review by austinlparker — APPROVED** [review]
couple of nits but overall very nice! I rather like the shuffling of the transaction subsection.

you tend to be a bit more terse than I am, but I think it works well here.

**Comment by bhs** [comment]
Thanks, @austinlparker! I renamed the chapter file and did a grep for "customers" throughout the CNO repo.

**Review by austinlparker — APPROVED** [review]

**Comment by austinlparker** [comment]
You can add those words that the spell checker is flagging to the .cspell.json file and it'll stop yelling about them

**Pull request merged**